packages/components/dropdown/dropdown.component.ts

```diff
@@ -146,7 +146,7 @@ export class KbqDropdown implements AfterContentInit, KbqDropdownPanel, OnInit,
     private _overlapTriggerY: boolean = this.defaultOptions.overlapTriggerY;
     private _hasBackdrop: boolean = this.defaultOptions.hasBackdrop;
 
-    triggerWidth: string = '';
+    triggerWidth: string;
```

Comment:
получается, что переменная может быть не определена?
```suggestion
    triggerWidth: string | undefined;
```

Reply:
`triggerWidth` может быть не инициализирована. Объединять string с undefined нельзя...

Напомни в понедельник обсудить

Reply:
get/set в данном случае должен решить эту проблему

Reply:
тут нет проблемы и гет/сет пока избыточны. Все обсудили, а ты опять за свое...

Reply:
получается, что мы пришли к разным выводам